depthai_nodes/ml/messages/img_detections.py

```diff
@@ -240,6 +240,44 @@ def keypoints(
             raise ValueError("Keypoints must be a list of Keypoint objects.")
         self._keypoints = value
 
+    def get_xyxy_bbox_points(self) -> NDArray[np.float32]:
+        """Returns the [x1, y1, x2, y2] bounding box points.
```

Comment:
If bbox is rotated this gives axis-alligned bbox right? If so maybe we want to explicitly mention "axis-alligned bounding box" in the docstring

Reply:
Yes true, it doesnt take into account the angle. Updated docstring in [c90782e](https://github.com/luxonis/depthai-nodes/pull/124/commits/c90782e683de48376b09392de1ad62db10e3bab4).